packages/devtools_app/lib/src/screens/inspector/diagnostics_node.dart

```diff
@@ -47,6 +47,10 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
     this.parent,
   );
 
+  /// Stores the [TextStyle] that was used when building the description. When
+  /// not set, then the description has not been built yet.
```

Comment:
blank line after first sentence in dart doc.

Also please describe why we need to store this value